src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.tsx

```diff
@@ -31,6 +31,9 @@ import { HttpStart, NotificationsStart, SavedObjectsStart } from '../../../../..
 import { AssociationDataSourceModalMode } from '../../../common/constants';
 import { Logos } from '../../../../../core/common';
 import { DirectQueryConnectionIcon } from '../workspace_form';
+import { DataConnectionIcon } from '../workspace_form';
```

Comment:
import this from line 33. also do we have multiple cases of defining icon throughout the app?

Reply:
Yes, this is still pending as I'm finding a better way to maintain different state between data-source, query connection, data-connection.